projects/adrv9009/src/devices/ad9528/ad9528.c

```diff
@@ -12,6 +12,9 @@
 
 #include "parameters.h"
 #include "ad9528.h"
+#ifndef ALTERA_PLATFORM
```

Comment:
nitpick: you have a ' in the commit title.

Reply:
It is correctly spelled, since I am referring to multiple drivers. Therefore the `'` comes at the end of the word.